### PR TITLE
inicialize Project

### PR DESCRIPTION
**Pull request by JavierAybar**
# BookStore `React` 📚 
- Use React Router.
- Use React components.

## Project requirements

- [ ] Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)

- [ ] Create a directory for your reusable components: `/components`.
- [ ] Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- [ ] [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of <App>
- [ ] Add 2 routes in your application.
- [ ] Create components inside of the `/components` directory:
- Contains individual book state, as well as a button to delete the book.
- Contains a list to renders individual books.
- Contains a form to create new books.
- Contains a navigation to link to the 2 routes you created.

### CODE REVIEW

File: src/pages/Books.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import '../style/Books.css';
+
+function Books() {
+  return (
+    <div>
+      <div className="books">
+        <Book />
+        <Book />
+        <Book />
+      </div>
```
Comment (abdulhamiid) [inline src/pages/Books.js]: According to the project requirements,

image

I suggest you create a `<BookList>` component that renders a list of books using the `<Book>` component. 🛠💪

File: src/pages/Books.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import AddBook from '../components/AddBook';
+import '../style/Books.css';
+
+function Books() {
+  return (
+    <div>
+      <div className="books">
+        <Book />
+        <Book />
+        <Book />
```
Comment (abdulhamiid) [inline src/pages/Books.js]: Kindly take the book data as a prop and renders the title and the author 👍

```JavaScript
<Book title="The Hunger Games" author="Suzanne Collins" />
```
`Props` is a special keyword in React, which stands for properties and is being used for passing data from one component to another similar to how the arguments are passed in a function. 🤗

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @JavierAybar 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Use React Router.
- ✔️ Use React components.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Efiamotu-1 — APPROVED** [review]
Hi @JavierAybar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

image

## Highlights :dart: 
* Correct Gitflow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**